libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,28 @@
+import { loadScript, getConfig } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+export default async function init() {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+
+  const OstCountry = 'US';
+  const OstWcsBaseUrl = 'https://wcs.adobe.io';
+  const OstWcsApiKey = 'wcms-commerce-ims-ro-user-milo';
+  const OstAosApiKey = 'wcms-commerce-ims-user-prod';
+  const OsLanguage = 'en';
+  const OstAosAccessToken = localStorage.getItem('AOS_ACCESS_TOKEN');
```

Comment:
I think we should keep this in session storage. We don't want this access token persisted over several sessions.

Reply:
Done ✅